laterpay/views/frontend/partials/post/gift/gift-pass.php

```diff
@@ -30,16 +30,36 @@
 if ( empty( $pass_id ) ) {
     return;
 }
+
+$subscription_data = LaterPay_Helper_Subscription::get_subscription_by_id( $pass_id );
+$is_subscription   = ( ! empty( $subscription_data ) && is_array( $subscription_data ) );
+
 ?>
 
-<div class="lp_js_giftCard lp_gift-card lp_gift-card-<?php echo esc_attr( $pass_id ); ?>">
+<div class="lp_js_giftCard lp_gift-card lp_gift-card-<?php echo esc_attr( $pass_id ); ?> <?php echo $is_subscription ? 'lp_gift-card-subscription' : 'lp_gift-card-timepass'; ?> ">
```

Comment:
```suggestion
<div class="lp_js_giftCard lp_gift-card lp_gift-card-<?php echo esc_attr( $pass_id ); ?> <?php echo $is_subscription ? 'lp_gift-card-subscription' : ''; ?> ">
```
is there a class for time pass card as well? could we make it empty if not?

Reply:
Removed.